authentik/providers/oauth2/views/github.py

```diff
@@ -90,7 +90,7 @@ def get(self, request: HttpRequest, token: RefreshToken) -> HttpResponse:
                 "created_at": "",
                 "updated_at": "",
                 "organization": {
-                    "login": "github",
+                    "login": slugify(request.tenant.branding_title),
```

Comment:
not sure if we should be changing this, do you know what this field is used for?